db/migrate/20240821124341_add_foreign_key_to_participant_outcome_api_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddForeignKeyToParticipantOutcomeAPIRequests < ActiveRecord::Migration[7.1]
+  def change
+    add_foreign_key :participant_outcome_api_requests, :participant_outcomes, primary_key: :ecf_id
```

Comment:
Im wondering if we need to link them via the `ecf_id` as opposed to the normal integer ids on the models 🤔  we should have ecf ids on both but link them via the normal ids maybe?

Reply:
@leandroalemao agree with abeer here, i don't think we should use `ecf_id` for primary key, its inconsistent with the rest of the app. 
We should see `ecf_id` as a separation leftover and maybe in the future will get removed.

Reply:
yeah agree tks guys!! Addressed in [6626487](https://github.com/DFE-Digital/npq-registration/pull/1624/commits/6626487038291e71b4dd5ce9460c6c559f662cb5) 👍 😄